src/remote_interface.h

```diff
@@ -0,0 +1,57 @@
+#ifndef REMOTE_INTERFACE_H_
+#define REMOTE_INTERFACE_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+#include "common.h"
+#include "constants.h"
+#include "remote_common.h"
+#include "uv.h"
+
+typedef struct _pow_remote_context PoW_Remote_Context;
+typedef struct _remote_impl_context RemoteImplContext;
```

Comment:
The naming convention seems different.

Reply:
Yes, they are dfferent, as refers to https://github.com/DLTcollab/dcurl/blob/develop/src/pow_avx.h#L11, https://github.com/DLTcollab/dcurl/blob/develop/src/implcontext.h#L10.

Reply:
Got it.
I guess we will need a consistent structure naming convention in the future.